README.md

```diff
@@ -1 +1,12 @@
-
+# Users
+CRUD implementation
+How to use methods:
+- _save(userObject)_, takes user object and saves it to the database, the id is adds automatically;
```

Comment:
Readme instructions should be used for describing how to install and use the application (not a Class). You shouldn't confuse javadocs with Readme.